test/fixtures/minimal/outputs.tf

```
@@ -34,19 +34,19 @@ output "memory_size_gb" {
   value = var.memory_size_gb
 }
 
-output "output_id" {
-  value = module.memstore.id
+output "output_ids" {
```

We could probably just pass through the entire outputs to simplify things:

```suggestion
output "outputs" {
  value = module.memstore
```

---

Got it